microprofile/WFLY-12758_integrate_opentracing.adoc

```diff
@@ -0,0 +1,108 @@
+= Integrate Microprofile OpenTracing 1.3
+:author:            Emmanuel Hugonnet
+:email:             ehugonne@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          microprofile,opentracing,observability,tracing,openshift
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Currently the integration of Microprofile Opentracing is minimal as you can't configure the tracer using the WildFly Management Model.
+So we will implement a way to define several tracer configurations (Jaeger) to be used in deployments.
+One of those configuration will become a default one so that deployments that don't define which configuration to use will use this one.
+In this proposal we will only support Jaeger as Tracer implementation but it should be able to be easily extendable to support other Tracers implementations:
+* Support https://github.com/eclipse/microprofile-opentracing/blob/1.3/spec/src/main/asciidoc/microprofile-opentracing.asciidoc[Eclipse MicroProfile OpenTracing 1.3] for applications deployed in WildFly, including JAX-RS and CDI integration. Passing the TCK is required.
+* Support a concrete tracer, such as Jaeger's.
+* Allow applications to supply their own tracers via `TracerResolver`.
```

Comment:
Let me suggest removing this sentence. In my opinion, the wording "own tracers" could be a little bit confusing - other tracer implementations are beyond this RFE scope as we guarantee Jaeger tracer configuration ability only.

Reply:
Hum, the support for TracerResolver was already in place, this would be a regression in terms of features over the previous releases. I'm talking tracer implementations here with their configuration being only supported by the user/TracerResolver mechanism.

Reply:
What are examples of own tracers users can supply ?

Reply:
@jstourac and @okotek you worked on opentracing stuff before [1], please comment on this topic

[1] https://github.com/wildfly/wildfly-proposals/pull/106 and https://github.com/wildfly/wildfly-proposals/pull/236

Reply:
some specific jaeger tracer implementation for example.